File: android/src/androidTest/java/org/ligi/survivalmanual/ScreenShotTaker.java
package org.ligi.survivalmanual;

import com.jraska.falcon.FalconSpoon;

class ScreenShotTaker {

    static void takeScreenShot(final MainActivity activity, final String help) {
        try {
            FalconSpoon.screenshot(activity, help);
        } catch (Exception ignored) {
            // we could not take the screenshot - no big deal
        }
    }
}

File: android/src/androidTest/kotlin/org/ligi/survivalmanual/ScreenShotTaker.kt
package org.ligi.survivalmanual

import com.jraska.falcon.FalconSpoon

object ScreenShotTaker {

    fun takeScreenShot(activity: MainActivity, help: String) {
        try {
            FalconSpoon.screenshot(activity, help)
        } catch (ignored: Exception) {
            // we could not take the screenshot - no big deal
        }
    }
}
